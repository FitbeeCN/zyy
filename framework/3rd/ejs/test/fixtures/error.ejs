<<<<<<< HEAD
<ul>
  <% if (users) { %>
    <p>Has users</p>
  <% } %>
=======
<ul>
  <% if (users) { %>
    <p>Has users</p>
  <% } %>
>>>>>>> 7af941ee074ba19b0302249f5332e62ee930056a
</ul>