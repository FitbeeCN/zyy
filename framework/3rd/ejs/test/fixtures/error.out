<<<<<<< HEAD
ReferenceError: error.ejs:2
    1| <ul>
 >> 2|   <% if (users) { %>
    3|     <p>Has users</p>
    4|   <% } %>
    5| </ul>

=======
ReferenceError: error.ejs:2
    1| <ul>
 >> 2|   <% if (users) { %>
    3|     <p>Has users</p>
    4|   <% } %>
    5| </ul>

>>>>>>> 7af941ee074ba19b0302249f5332e62ee930056a
users is not defined